client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/OutboundLineEditForm.tsx

```diff
@@ -79,13 +84,16 @@ export const OutboundLineEditForm: React.FC<OutboundLineEditFormProps> = ({
 
   const unit = item?.unitName ?? t('label.unit');
 
-  const updateIssueQuantity = (quantity: number) => {
-    setIssueQuantity(
-      Math.round(
-        quantity / Math.abs(Number(packSizeController.selected?.value || 1))
-      )
-    );
-  };
+  const updateIssueQuantity = useCallback(
```

Comment:
It takes me a long time to internalise why a memoized function is also needed here.

I'm slightly fearful of needing to make changes here and it getting very complicated.

Maybe this is a pretty typical scenario? Maybe it is just me? I'm curious if it could be refactored somehow.